src/Gui/DlgSettings3DView.ui

```diff
@@ -762,31 +689,45 @@ bounding box size of the 3D object that is currently displayed. </string>
           </property>
          </widget>
         </item>
-        <item row="1" column="0">
+        <item row="4" column="0">
+         <widget class="QLabel" name="orbitLabel">
+          <property name="text">
+           <string>Orbit style</string>
+          </property>
+         </widget>
+        </item>
+        <item row="14" column="3">
          <widget class="QLabel" name="backlightLabel">
+          <property name="enabled">
+           <bool>false</bool>
+          </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
-           <string>Intensity of backlight</string>
```

Comment:
Why remove this descriptive part of the tooltip ? Isn't `Intensity` too generic ?

Reply:
> Why remove this descriptive part of the tooltip ? Isn't `Intensity` too generic ?

Because it was requested in the forum: https://forum.freecadweb.org/viewtopic.php?f=17&t=43984#p375026
and I agreed that it is clear because this widget only becomes enabled when there is a backlight color.